app/views/dashboard/_support_source.html.erb

```diff
@@ -1,36 +1,43 @@
-        <a href="javascript:void(selectNone(<%= support_source.id %>))" class="tool">Select none</a>
-        <a href="javascript:void(ignoreSelectedTickets(<%= support_source.id %>))" class="tool">Ignore selected</a>
+        <a href="javascript:void(selectAll(<%= support_source[:id] %>))" class="tool">Select all</a>
+        <a href="javascript:void(selectNone(<%= support_source[:id] %>))" class="tool">Select none</a>
+        <a href="javascript:void(ignoreSelectedTickets(<%= support_source[:id] %>))" class="tool">Ignore selected</a>
     </div>
 
-    <% if support_source.scheduler.working? %>
-        <div class="support-source-announcement">
-            <div class="alert alert-info">This support source is being synchronized. <a href="javascript:void(window.location.reload())" class="alert-link">Reload this page</a> in a few minutes to check for updates.</div>
-        </div>
-    <% end %>
 
-    <% if support_source.tickets.empty? %>
-        <div class="intro-text intro-text-with-margin">
-            <p>Hurray, there are no open support tickets for this source. :-)</p>
-        </div>
-    <% else %>
+    <div id="tickets_list">
+    <% if support_source[:tickets] %>
+        <ul class="list-unstyled">
+            <%= render partial: 'ticket', collection: support_source[:tickets] %>
+        </ul>
+     <% else %>
+        <% if support_source.scheduler.working? %>
+            <div class="support-source-announcement">
+                <div class="alert alert-info">This support source is being synchronized. <a href="javascript:void(window.location.reload())" class="alert-link">Reload this page</a> in a few minutes to check for updates.</div>
+            </div>
+        <% end %>
         <ul class="list-unstyled">
             <%= render partial: 'ticket', collection: support_source.tickets %>
         </ul>
-    <% end %>
-
-    <div class="support-source-footer">
-        <%= link_to admin_area_path do %>
-            Last synchronized at:
-            <% if support_source.scheduler.last_run_time %>
-                <%= distance_of_time_in_words_to_now(support_source.scheduler.last_run_time) %> ago
-            <% else %>
-                N/A
+        <div class="support-source-footer">
+            <%= link_to admin_area_path do %>
+                <% if support_source.scheduler.working? %>
+                    <%= button_to 'Synchronize now', '#', disabled: true, class: 'btn btn-primary' %> <small>scheduler is working now</small>
+                <% else %>
+                Last synchronized at:
+                    <% if support_source.scheduler.last_run_time%>
+                        <%= distance_of_time_in_words_to_now(support_source.scheduler.last_run_time) %> ago
+                    <% else %>
+                        N/A
+                    <% end %>
+                    <br />
+                    <%= link_to 'Sync now', "admin/sync_#{support_source.type_name.downcase}", method: 'post', class: 'btn btn-primary' %>
```

Comment:
Using `support_source.type_name.downcase` here instead of `support_source.name.downcase`. In the case of RSS the name can be user defined.